mlir/utils/jenkins/Jenkinsfile.migraphxintegration

```diff
@@ -1,231 +1,317 @@
 boolean shouldRunOnMachine(String machine){
     switch (machine){
-        case "rocm-framework-38.amd.com":
-            return (params.MI200 == true)
-        case "aus-navi3x-13.amd.com":
+        case "ixt-hq-ubb4-31":
```

Comment:
Instead of a mi200 machine that has a gpu problem. Omkar gave me a mi250 machine to use. Currently busy with another CI so I'm waiting to finish the job, and run the CI to test the models.